gm4_crossbow_cartridges/data/gm4_crossbow_cartridges/functions/main.mcfunction

```diff
@@ -1,4 +1 @@
-#check for arrow that places a torch or crop
-execute as @e[type=minecraft:arrow,tag=gm4_cb_torch,nbt={inGround:1b}] at @s run function gm4_crossbow_cartridges:torch/torch_check
 
```

Comment:
Since `main` is empty now, it can be removed. You can also remove the two schedule commands in `load` and `init`